doc/TACACSPLUS_PASSKEY_ENCRYPTION.md

```diff
@@ -41,60 +42,62 @@ The revised data handling procedure among the modules is outlined as follows:
            |           |   
 +----------v-----------v---------+                                      +---------------------+
 | AUTHENTICATION                 |                                      |                     |
-|   +-------------------------+  |  Decrypted passkey                   |  +------------+     |
-|   | PAM Configuration Files <-----------------------------------------+  | AAA Config |     |
-|   +-------------------------+  |                                      |  +------------+     |
-|                                |                                      |                     |
-|         +-------------+        |                                      |  HostCfg Enforcer   |
-|         |     PAM     |        |                                      +----------^----------+
-|         |  Libraries  |        |                                                 |
-|         +-------------+        |                                                 | Encrypted passkey
-+---------------+----------------+                                                 |
-                |                                                                  |
-           +----v----+                                                     +-------+--------+
-           |         |             Encrypted passkey                       |                |
-           |   CLI   +------------------------------------------------------>    ConifgDB   |
-           |         |                                                     |                |
+|   +-------------------------+  |         Decrypted passkey            |  +------------+     |
+|   | PAM Configuration Files    <------------+       +-----------------+  | AAA Config |     |
+|   +-------------------------+  |            |       |                 |  +------------+     |
+|                                |            |       |                 |                     |
+|         +-------------+        |         +-------------+              |  HostCfg Enforcer   |
+|         |     PAM     |        |         |             |   key-store  +----------^----------+
+|         |  Libraries  |        |    +---->  Master key |     __                  |
+|         +-------------+        |    |    |   Manager   |----|r |                 | Encrypted passkey
++---------------+----------------+    |    |             |    |o |                 |
+                |                     |    +-------------+    |o |                 |
+           +----v----+                |           |           |t |         +-------+--------+
+           |         |                |           |            --          |                |
+           |   CLI   +----------------+           +------------------------>    ConifgDB    |
+           |         |                                  Encrypted passkey  |                |
```

Comment:
Fix the typo. ConifgDB

Reply:
Fixed it. @venkatmahalingam can you merge this PR? or should I ask @zhangyanzhao ?